src/meta/meta.c

```diff
@@ -91,12 +92,10 @@ const char *MetaData_ToString(MetaData *metadata)
 {
     assert(metadata);
 
```

Comment:
I like the impelment of the left one, which is at first to filter out the exception flow with invalid argument feed.